3.10/installation-mac-osx.md

```diff
@@ -96,9 +97,14 @@ also need to update Homebrew executing the command `brew update`.
 Package Installation
 --------------------
 
-We provide a command-line app called *ArangoDB-CLI*.
-Visit the official [Download](https://www.arangodb.com/download){:target="_blank"} page of the
-ArangoDB website and download the *DMG* Package for macOS.
+ArangoDB provide a command-line app called *ArangoDB-CLI*.
```

Comment:
validate notarization of the respective executeables like this:
https://github.com/arangodb/release-test-automation/blob/main/release_tester/arangodb/installers/base.py#L99